package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "memcached",
-  "version": "2.2.2",
+  "version": "2.2.3",
```

Comment:
According to [semver](https://semver.org/), we should change the MINOR version. Otherwise LGTM. 

In addition, Travis CI config had some problems before, we have fix it. Please try to commit again. :)

Reply:
Oh...I'm sorry to hear that. Because of that CI problem, I tried several times but it didn't work.
It's great that you can continue to maintain this project! Thank you~